locust/web.py

```diff
@@ -165,7 +166,7 @@ def send_assets(path):
 
             return send_from_directory(directory, path)
 
-        @app.route("/")
+        @app.route(f"{self.base_url}/")
```

Comment:
If `self.base_url` is `/` (as it is by default) we will end up with `@app.route("//")` right?